app/forms/validation/mappings/MappingUtil.scala

```diff
@@ -40,10 +40,11 @@ object MappingUtil extends Formatters {
   def currency(requiredKey: String,
                wrongFormatKey: String = "common.error.invalid_currency_format",
                maxAmountKey: String = "common.error.amountMaxLimit",
+               maxAmountValue: BigDecimal,
```

Comment:
We should keep parameters without defaults before the ones that do have defaults. Either this field needs to be moved up, or you can add a default value.